connector/src/main/scala/quasar/qscript/MapFunc.scala

```diff
@@ -221,10 +221,21 @@ object MapFunc {
   }
 
   def normalize[T[_[_]]: BirecursiveT: OrderT: ShowT, A: Show]
-      : CoEnv[A, MapFunc[T, ?], FreeMapA[T, A]] => CoEnv[A, MapFunc[T, ?], FreeMapA[T, A]] =
+      : CoMapFuncR[T, A] => CoMapFuncR[T, A] =
     (repeatedly(rewrite[T, A]) _) ⋘
       orOriginal(foldConstant[T, A].apply(_) ∘ (const => rollMF[T, A](Constant(const))))
 
+  def thingy[T[_[_]]: BirecursiveT](left: Symbol, right: Symbol)
+      : CoMapFuncR[T, JoinSide] => CoMapFuncR[T, JoinSide] = {
+    val leftName: String = left.toString + "_hello"
+    val rightName: String = right.toString + "_hello"
```

Comment:
Can we just use the name created in the join condition and omit the special identifier? Is this analogous to how we already assume we won't hit conflicts with tmp vars we create in LP?